planning/scenario_planning/lane_driving/behavior_planning/behavior_velocity_planner/launch/behavior_velocity_planner_tmp.launch

```diff
@@ -0,0 +1,53 @@
+<launch>
```

Comment:
Why did you create tmp.launch? I guess "beahvior_velocity_planner.launch.xml" is an appropriate launch file name.

Reply:
Because vehicle_param_file come from planning simulator.launch so it cant test alone

Reply:
I guess you can test as below.
`ros2 launch behavior_velocity_planner behavior_velocity_planner.launch.xml vehicle_param_file:=(vehicle info yaml path)`

Reply:
1688ea6